bootstrap/kubeadm/types/utils_test.go

```diff
@@ -78,12 +79,20 @@ func TestKubeVersionToKubeadmAPIGroupVersion(t *testing.T) {
 			want:    v1beta2.GroupVersion,
 			wantErr: false,
 		},
+		{
+			name: "pass with kubernetes version for kubeadm API v1beta3",
```

Comment:
Let's also add a test "pass with minimum kubernetes version for kubeadm API v1beta3" (v1.22.0),